1077/en.md

```diff
@@ -15,7 +17,7 @@ But what's the proof they lie on the segment **AB** and there can't be any other
 See: https://math.stackexchange.com/questions/628117/how-to-count-lattice-points-on-a-line
 
 ## Complexity
-- Time Complexity: O(T).
+- Time Complexity: O(T * lg(N)). Where N = **max(a, b)** of **gcd(a, b)**. [Check](https://www.quora.com/What-is-the-time-complexity-of-Euclids-GCD-algorithm/answer/M00nlight-2?ch=10&oid=16747157&share=a62b29f9&srid=xek7a&target_type=answer) for the time complexity of Eulclid's GCD algorithm.
```

Comment:
Let's not point to Quora for references. They often lock answers. This might work as well: https://stackoverflow.com/questions/3980416/time-complexity-of-euclids-algorithm.

Additionally, typo on "Euclid".

Reply:
done!